data-models/spaces/schema.json

```diff
@@ -22,7 +22,7 @@
         "maximum": 2
       }
     },
-    "minutesSinceLastOccupied": {
+    "minutesSinceLastOccupancy": {
```

Comment:
I prefer minutesSinceLastOccupied

Reply:
I understood something wrong the in an other pr